src/git/config.go

```diff
@@ -141,9 +141,7 @@ func GetRemoteUpstreamURL() string {
 // GetURLHostname returns the hostname contained within the given Git URL.
 func GetURLHostname(url string) string {
 	hostnameRegex, err := regexp.Compile("(^[^:]*://([^@]*@)?|git@)([^/:]+).*")
-	if err != nil {
-		log.Fatal("Error compiling hostname regular expression: ", err)
-	}
+	logs.FatalOn(err, "Error compiling hostname regular expression: ", err)
```

Comment:
We can definitely provide helper methods that wrap errors, that's a great idea. I don't see how this would save us from having to provide the error object twice here, though. Here is how I think this piece of code would look like with your proposed set of methods:

```go
logs.FatalOnWrapf(err, "Error compiling hostname regular expression: %s", err)
```

I don't want to introduce wrapping errors in this change, but merely dry up the code structure here. We can introduce wrapping in a follow-up PR.